astropy/table/serialize.py

```diff
@@ -7,6 +7,7 @@
 from .column import Column
 from .table import Table, QTable, has_info_class
 from astropy.units.quantity import QuantityInfo
+from astropy.coordinates import representation as coorep
```

Comment:
After much flailing I finally figured out that this line is what is breaking things and causing all the confusing problems for me with warnings and test fails and leap seconds re-downloading every time.  It is tricky because of circular imports, so the problem only shows up in a certain import order.  You should be able to reproduce with a single `import astropy.io.ascii` without anything first.  But `import astropy.coordinates; import astropy.io.ascii` is fine.  And `import astropy.table` first gives `/Users/aldcroft/git/astropy/astropy/table/column.py:1020: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  result = getattr(super(), op)(other)`

This is latter warning is from the leap second stuff, and goes away if I remove this line here in `serialize.py`.

Reply:
Darn, that must be because it imports `Time` - I'm sorry that I've caught all this mess, @pllim already had to move other imports inline to get things to pass.

Reply:
I suggest a quick workaround to hardcode as strings all the representations currently in astropy.

Reply:
OK, will make a PR. Clearly trying to be clever comes back to haunt oneself...

Reply:
See #10282